### ISSUE TITLE
MobileVit overfit

### ISSUE THREAD

**Issue by Jerryme-xxm** [kentaroy47/vision-transformers-cifar10#22]
Thanks for your great work!

I add a Vit model named mobilevit and trained it, but the val accuracy is limited to 0.82, while train accuracy reach 0.99.
To avoid overfit, I set weight_decay to 1e-3, and the val accuracy is limited to 0.89, still very low.

I have tried other training code, which sets lr=0.1 and use sgd optimization, the val accuracy can reach 0.92.

May I ask how to improve this situation?

**Comment by kentaroy47** [comment]
can you share the mobilevit implementation?
